moneyadviceservice/frontend
# [DEV APPROVED] 8666 Use mas-cms-client gem for home page

**Target Process ticket**
[8666](https://moneyadviceservice.tpondemand.com/entity/8666-use-mas-cms-client-gem-on)

**Summary**
Replaces legacy code for loading the Home Page and Home Page Preview from CMS with the mas-cms-client gem. Removes unused legacy code relating to the home page.

**QA / Testing**
Ensure the homepage loads correctly with all correct components in both English and Welsh.
Ensure the homepage loads correctly when previewing.

**Checklist**
